poll/public/handlebars/poll_results.handlebars

```diff
@@ -0,0 +1,43 @@
+<script id="poll-results-template" type="text/html">
+    <h3 class="poll-header">{{display_name}}</h3>
+    <div class="poll-question-container">{{{question}}}</div>
+    <ul class="poll-answers-results poll-results">
+    {{#each tally}}
+        <li class="poll-result">
+            <div class="poll-result-input-container">
+              <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked="True"{{/if}} />
```

Comment:
Where can the updated code be viewed? When should I take another look?
Finally, let me know when the sandbox is updated. Thanks :)

On Thu, Jun 4, 2015 at 3:28 PM, Sven Marnach <notifications@github.com>
wrote:

> In poll/public/handlebars/poll_results.handlebars
> <https://github.com/mckinseyacademy/xblock-poll/pull/13#discussion_r31756290>
> :
>
> > @@ -0,0 +1,43 @@
> > +<script id="poll-results-template" type="text/html">
> > +    <h3 class="poll-header">{{display_name}}</h3>
> > +    <div class="poll-question-container">{{{question}}}</div>
> > +    <ul class="poll-answers-results poll-results">
> > +    {{#each tally}}
> > +        <li class="poll-result">
> > +            <div class="poll-result-input-container">
> > +              <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked="True"{{/if}} />
>
> Done.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mckinseyacademy/xblock-poll/pull/13/files#r31756290>.
>